package.json

```diff
@@ -17,7 +17,9 @@
     "jest": "NODE_ICU_DATA=`yarn -s run node-full-icu-path` jest",
     "test": "eslint ./ && flow check && yarn test:ts && yarn jest",
     "test:ts": "tslint -p tsconfig.json",
-    "deploy": "yarn next build && yarn next export && gh-pages -d out",
+    "copy-sandbox-examples": "mkdir -p ./out/pages && find ./pages -type d -mindepth 1 -maxdepth 1 -exec cp -R \\{\\} ./out/pages \\;",
```

Comment:
Maybe rename to codesandboxes then?

Reply:
I meant rename dist folder to codesandboxes. `pages` doesn't make much sense in gh-pages

Reply:
:-) done